src/components/Icon/IconBase.js

```diff
@@ -10,22 +10,25 @@ const { block, elem } = bem({
     propsToMods: ['context', 'margin']
 });
 
-const adjustSize = size => {
+const adjustSize = (size, keepWidth) => {
```

Comment:
Maybe it makes more sense to have something like `preserveAspectRatio` instead?

Reply:
great idea